<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainDialog</class>
 <widget class="QDialog" name="mainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/image/receive_2.png</pixmap>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
background-color: rgb(110, 158, 207);
color: rgb(255, 255, 255);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>262</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="l_port_2">
            <property name="text">
             <string>端口：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_port"/>
          </item>
          <item>
           <widget class="QToolButton" name="btn_modify">
            <property name="text">
             <string>确定/修改</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QToolButton" name="btn_startServe">
          <property name="text">
           <string>启动服务</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_stopServe">
          <property name="text">
           <string>停止服务</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="l_serverstate_2">
        <property name="text">
         <string>服务器已经启动，等待连接中…</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_connectList_2">
        <property name="text">
         <string>连接列表：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="lv_connectList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>161</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <widget class="QLabel" name="l_needRecognitize_2">
        <property name="text">
         <string>需要人工识别图像队列：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="lv_needRecognize"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <item>
       <widget class="QLabel" name="l_recognized_2">
        <property name="text">
         <string>已人工识别图像队列：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="lv_recognized"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>221</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_16">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_17">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="l_queryWay_2">
          <property name="text">
           <string>查询方式：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_queryWay">
          <item>
           <property name="text">
            <string>按身份证号查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按车牌号查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按姓名查询</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="l_queryContent_2">
        <property name="text">
         <string>请输入要查询的内容：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLineEdit" name="le_query">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_query">
          <property name="text">
           <string>确认查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <item>
       <widget class="QLabel" name="l_infoList_2">
        <property name="text">
         <string>违章车辆信息列表：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_info_2">
        <property name="text">
         <string>违章时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tv_infoList">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="btn_min">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>0</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
background-color: rgb(110, 158, 207);
color: rgb(255, 255, 255);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
